packages/projects-docs/next.config.js

```diff
@@ -21,7 +21,7 @@ module.exports = withTM(
         */
         {
           source: "/",
-          destination: "/learn/introduction/overview",
+          destination: "/learn",
```

Comment:
We should also check that we have dynamic redirects in place for links that changed because of this base path update.

Reply:
I think that is the intention of this doc. We need to add the deleted urls and specify the redirections here

Reply:
I added all of the links that will be outdated due to this change at the bottom of this file